about/get-wpe.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: page
+title: "Get WPE WebKit"
+tags: [about]
+data: { dateless: "true" }
+permalink: /about/get-wpe.html
+sitemapChangeFrequency: yearly
+resources: [
+	{
+		"title": "Package Releases",
+		"url": "/release/",
+		"text": "The most recent stable and development releases of WPE codebases."
+	},
+	{
+		"title": "Release Schedule",
+		"url": "/release/schedule/",
+		"text": "How often does WEPWebKit release, and when?"
+	},
+	{
+		"title": "Security Advisories",
+		"url": "/security/",
+		"text": "A complete archive of our security adviories, often with download links."
+	},
+	{
+		"title": "Performance Tips",
+		"url": "https://github.com/Igalia/meta-webkit/wiki/PerformanceTips",
+		"text": "A few pointers on how to get better performance out of WPEwebkit. (Github Wiki)"
+	}
+]
+---
+
+<header class="page">
+
+# Get WPE
+
+</header>
+
+WPE is an open source/free software project. That means that you can
+get the source code directly, study it, modify it, and deploy it to
+serve your needs. While this can be an involved process, there
+are different ways to get your hands on WPE, depending on your
+needs.
+
+## Before starting
+
+Before getting the code, it's a good idea to be familiar with what you
```

Comment:
My only real comment here is that are we sure about this particular organization of the page?  That is: Do you need to know any of this if you're here to get a distro?  I don't think so, right?  I guess it doesn't hurt but it implies you might need to install something else and I think in that case, or in the balena case (if we keep it) you don't?

Reply:
Well, you have to install several packages if you're going the distro way. If you don't know that, you might just try to find the "one" WPE package, which is not guaranteed to exist.

If you are going to use a more involved setup, you should also have a vague idea of what packages you are going to be dealing with. I don't see any scenario involving getting your hands on WPE where knowing this will not be useful.

Reply:
In the WPE Commission meeting @bkardell to probably move the "Pick your flavour" part to the top, and then maybe have 3 giant groups to separate between then and properly separate things (e.g. with different backgrounds per block). Maybe doing that would address @bkardell comments here?